web/docs/integrations/github.md

```diff
@@ -6,9 +6,60 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 # GitHub Integrations
```

Comment:
Maybe change the title from "GitHub Integrations" to just "GitHub". (same goes for Google page)

<img width="358" alt="Screenshot 2023-01-30 at 16 04 02" src="https://user-images.githubusercontent.com/2223680/215513782-1b420d6f-fa3b-4cec-98d4-931434258056.png">